pkg/machines/components/diskAdd.jsx

```diff
@@ -111,31 +114,27 @@ const PermanentChange = ({ idPrefix, onValueChanged, dialogValues, provider, vm
     }
 
     return (
-        <div className="row">
-            <div className="col-sm-1 dialog-field" />
-
-            <div className="col-sm-11 dialog-field add-disk-attach-perm">
+        <FormGroup>
+            <Grid.Col componentClass={Form.ControlLabel} sm={2}>
+                {_("Attach permanently")}
```

Comment:
This ends up looking a bit off, since it doesn't fit the width, so it does the line break.
Currently the label is on the right of the checkbox, and I think it works better there.

Reply:
ok, I 'll put it back.

Reply:
@andreasn the attach permanently string should be bold like used for labels or have it different from labels like it was in the past?

Reply:
non-bold I think is most correct